build.gradle

```diff
@@ -1,6 +1,8 @@
 plugins {
     id 'java'
     id 'war'
+    id 'net.ltgt.apt' version '0.10'
```

Comment:
what is it?

Reply:
That was in instruction to add Lombock plugin to the project, but I can see it is not necessary. So I will remove it.

Reply:
Removed.